infra/bootstrap/index.ts

```diff
@@ -54,6 +55,38 @@ export = async () => {
     }],
   });
 
+  // Create a role that the log fetcher Lambda can assume to list tags on Cloudwatch alarms in the current account
+  const alarmTagsRole = new aws.iam.Role(`${prefix}alarmListTagsRole`, {
+    name: `${prefix}alarmListTagsRole`,
```

Comment:
Why don't we just grant the lambda execution role the permissions to read tags, rather than creating a new role and assuming it in the lambda?

Reply:
Does that work cross-account though? I had the same thought initially but couldn't get it to access the resources without assuming a role in the other accounts.